hw_diag/tasks.py

```diff
@@ -30,6 +31,7 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['last_updated'] = now.strftime("%H:%M UTC %d %b %Y")
 
     get_ethernet_addresses(diagnostics)
+    get_temperatures(diagnostics)
```

Comment:
Can you rebase on https://github.com/NebraLtd/hm-diag/pull/337 and convert this into a Diagnostic?

Reply:
Sorry, rebasing on #337 doesn't make sense. But please look at the DiagnosticReport pattern that is being introduced there and copy here.

Reply:
I have almost zero Python skills and could not follow the process